### DISCUSSION TITLE
Buscador predictivo

### DISCUSSION THREAD

**Discussion by GraceFinlay** [Telefonica/mistica-design#1019] [Proposals]
Buenas tardes,

Mi nombre es Grace y estoy trabajando en el proyecto de ecommerce global y os voy a indicar una serie de dudas relacionadas con Mística que han surgido en el desarrollo del proyecto.

# Contexto

El proyecto que está en marcha quiere incorporar unas mejoras en la tienda para que diferentes usuarios puedan comprar diferentes dispositivos. Estas mejoras que se quieren introducir son entre otras la mejora de los flujos de usuario y diferentes funcionalidades.

# Problema

Para este proyecto se requiere utilizar Mística y observamos que tenemos unas necesidades concretas en cuanto a los componentes. Para nosotros es importante obtener feedback de vuestra parte con el fin de entender cómo podríamos implementar el proyecto usando Mística y qué cubra las necesidades planteadas.En este caso queremos diseñar una card moderna, limpia y con la información necesaria para el usuario.

# Propuesta
![Buscador](https://user-images.githubusercontent.com/99174134/214347099-6125b9dc-c3f7-4bec-b93b-748c160903a8.png)

Para el buscador necesitamos crear las búsquedas predictivas como se muestra en los ejemplos anteriores. 

No encontramos nada similar en Mística. 

Estamos abiertos a cualquier feedback que nos podáis dar para abordar el proyecto.

De antemano muchas gracias.

**Comment by yceballost** [comment]
Entiendo que sería muy parecido a esto
https://mistica-web.vercel.app/?path=/story/m%C3%ADstica-lab-autocomplete--default

**Reply by GraceFinlay** [reply]
Gracias de nuevo : )
Sí, veo que habéis unido  el input field, un botón y un dropdown. En este caso el botón no sería necesario pero el dropdown si y es lo que no logro ver en los assets. Podría utilizar para esto el input field: search/opened pero cuando selecciono esta opción se cambia a select/opened.

**Reply by yceballost** [reply]
ah vale, entiendo que quieres que exista la opción en Figma para desplegar el dropdown predictivo en el input search

lo añado como tarea y lo incluimos en cuanto podamos

**Reply by yceballost** [reply]
añadido como tarea, intentamos sacarlo para la siguiente release
https://github.com/Telefonica/mistica-design/issues/1024
